### PR TITLE
SPCTR-2685 Fix: Typography - Fonts were not loading inside of edit template iframe.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2685

### Screenshots

- https://share.getcloudapp.com/6qubb0rZ

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

File: src/components/typography/fontloader.js [hunk 1]
```diff
@@ -59,41 +59,21 @@ const WebfontLoader = ( props ) => {
 			} );
 			addFont( props.config.google.families[ 0 ] );
 		}
-		const siteEditTheme = document.getElementsByClassName( 'edit-site' );
+		const checkIsIframe = document.getElementsByTagName( 'iframe' );
```
Comment (supriyajjadhav) [inline src/components/typography/fontloader.js:62]: Can we directly select document.getElementsByTagName( 'iframe' )[ 0 ] ?

Comment (AbhijitNage123) [reply src/components/typography/fontloader.js:62]: Ther might be case happen Iframe is not detected. So, keeping it same.

Comment (AbhijitNage123) [reply src/components/typography/fontloader.js]: Resolved

Comment (supriyajjadhav) [reply src/components/typography/fontloader.js]: const checkIsIframe = document.getElementsByTagName( 'iframe' )[0];
if ( checkIsIframe )

**Review by supriyajjadhav — APPROVED** [review]

File: src/components/typography/fontloader.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/components/typography/fontloader.js]: Push the changes

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**